package.json

```diff
@@ -62,6 +62,7 @@
     "diacritics": "^1.2.2",
     "dompurify": "^0.8.4",
     "dotenv": "^4.0.0",
+    "electron": "^1.7.8",
```

Comment:
Actually, can you do `yarn add electron@1.7.8 --dev` so we can keep a dev dependency otherwise the native extension might cause problems in Heroku.

Reply:
Oops yeah good catch!